ports/casadi/vcpkg.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "casadi",
+  "version": "3.6.7",
+  "description": "CasADi is a symbolic framework for numeric optimization implementing automatic differentiation in forward and reverse modes on sparse matrix-valued computational graphs. It supports self-contained C-code generation and interfaces state-of-the-art codes such as SUNDIALS, IPOPT etc. It can be used from C++, Python or Matlab/Octave.",
+  "homepage": "http://casadi.org",
```

I see that this is the URL they publish but ... should we prefer the actual https site instead?
```suggestion
  "homepage": "https://web.casadi.org/",
```
(I don't know what the vcpkg owners prefer.)

---

I'm equally unaware of the preferences here. From the casadi point of view, `casadi.org` is the perpetual place to be. And `web.casadi.org` just the current incarnation of the site. However, I have noticed that some corporate laptops block the redirect of `casadi.org` to `web.casadi.org` so the latter has its merits too.

---

Lets use https://web.casadi.org

---

Okay, adapted.